diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -493,9 +547,18 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
                 if (node.treePrev.isWhiteSpace()) {
                     node.treeParent.removeChild(node.treePrev)
                 }
-                val newLineNodeOnPreviousLine = node.findAllNodesWithConditionOnLine(node.getLineNumber() - 1) {
-                    it.elementType == WHITE_SPACE && it.textContains('\n')
-                }?.lastOrNull()
+
+                val newLineNodeOnPreviousLine = if (node.treeParent.elementType == PROPERTY) {
```

Comment:
lets document this `if` branch too
its kind of hard to understand what it is `node.treeParent.treeParent....` in case of `property` type

Reply:
Done